brain/circuit.cpp

```diff
@@ -215,6 +230,12 @@ class MVD3 : public Circuit::Impl
         , _circuit( config.getCircuitSource().getPath( ))
     {}
 
+    size_t getNumNeurons() const final
+    {
+        // Dirty cast until ::MVD3::MVD3File::getNbNeuron becomes const
```

Comment:
I spoke to Adrien and he said this was already working on it. Still, I guess I can create an issue to make sure this does not get forgotten.